ambition_utils/rrule/models.py

```diff
@@ -304,3 +351,44 @@ def generate_dates_from_params(cls, rrule_params, time_zone=None, num_dates=20):
         rule = cls(rrule_params=rrule_params, time_zone=time_zone)
 
         return rule.generate_dates(num_dates=num_dates)
+
+    @classmethod
```

Comment:
do these two class methods need to be a class methods if you always have to pass a source rrule? can it just be a regular method so we call `rrule.clone()` and `rrule.clone_with_offset()`?

Reply:
I see no reason why they cannot be. I can't even justify why I made them that way.

Reply:
360a427